controllers/pwas/crud.js

```diff
@@ -187,7 +149,7 @@ router.get('/:pwa', function get(req, res, next) {
  * Delete a PWA.
  */
 router.get('/:pwa/delete', function _delete(req, res, next) {
-  getModel().delete(PWA, req.params.pwa, function(err) {
+  pwaModel.delete(req.params.pwa, function(err) {
```

Comment:
nit: you've used "=>" to declare a function above, decide on one pattern and use it consistently.